providers/tests/standard/utils/test_python_virtualenv.py

```diff
@@ -77,7 +74,7 @@ def test_should_create_virtualenv_with_system_packages(self, mock_execute_in_sub
         )
         assert "/VENV/bin/python" == python_bin
         mock_execute_in_subprocess.assert_called_once_with(
-            [sys.executable, "-m", "virtualenv", "/VENV", "--system-site-packages", "--python=pythonVER"]
+            ["pythonVER", "-m", "venv", "/VENV", "--system-site-packages"]
```

Comment:
```suggestion
            ["python", "-m", "venv", "/VENV", "--system-site-packages"]
```

Reply:
No, no - this is OK. pythonVER is passed there in tests :)

Reply:
```
venv_directory="/VENV", python_bin="pythonVER", system_site_packages=True, requirements=[]
```

The thing there is that we cannot specify `--python` in venv because it can only create a venv for it's own python version - so rather than  passing `--python` we have to run it via `pythonVER` interpreter :)